src/Test/Perf/ConsumptionParser/Program.cs

```diff
@@ -0,0 +1,134 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Roslyn.Test.Performance.Utilities.ConsumptionParser
+{
+    internal class Program
+    {
+        public static void Main()
+        {
+            const string prefix = "PerfResults-";
+            const string share = @"\\mlangfs1\public\basoundr\PerfTraces\";
+
+            var runs = new List<Tuple<string, ConsumptionParse, RunInfo>>();
+
+            // Collect run information from the share
+            foreach (var directory in Directory.GetDirectories(share))
+            {
+                var lastPath = directory.Substring(share.Length);
+                if (!lastPath.StartsWith(prefix))
+                {
+                    continue;
+                }
+
+                var date = lastPath.Substring(prefix.Length);
+                var consumptionXml = Path.Combine(directory, "ConsumptionTempResults.xml");
+                var resultJson = Directory.EnumerateFiles(directory, "Roslyn*.json").Single();
```

Comment:
Again, this should probably be a command line arg (although we could default to this value)